src/DevTools/setupTestWrapper.tsx

```diff
@@ -137,18 +131,30 @@ export const setupTestWrapperTL = <T extends OperationType>({
         )
       })
 
-      const operationName = operation?.request.node.operation.name
-      const operationVariables = operation?.request.variables
+      const operationName = operation.request.node.operation.name
+      const operationVariables = operation.request.variables
 
       return { operation, operationName, operationVariables }
     }
 
     const mockRejectLastOperation = (error: Error) => {
+      const operation = env.mock.getMostRecentOperation()
+
       act(() => {
-        env.mock.rejectMostRecentOperation(error)
+        env.mock.reject(operation, error)
       })
+      const operationName = operation.request.node.operation.name
+      const operationVariables = operation.request.variables
+
+      return { operation, operationName, operationVariables }
     }
 
+    const view = render(<TestRenderer />)
+
+    env.mock.resolveMostRecentOperation(operation => {
+      return MockPayloadGenerator.generate(operation, mockResolvers)
+    })
```

Comment:
Changes to the RTL setup:
- `mockRejectLastOperation()` returns the operation and its name/vars just like resolve version does
- operation does not need safe navigation operator
- (minor) render view after defining helpers